packages/connect-query/package.json

```diff
@@ -11,14 +11,12 @@
   "type": "module",
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
-      "import": "./dist/index.js",
-      "require": "./dist/index.cjs"
+      "module": "./dist/index.js",
```

Comment:
I suggest we merge this first, then merge it into https://github.com/connectrpc/connect-query-es/pull/180.
Feel free to commandeer that PR.